package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "frankie-barrios-ux-iop-1",
+  "version": "1.0.0",
+  "description": "UX Team Intern Onboarding Project",
+  "main": "index.js",
```

Comment:
won't need to define `main`, for the time being anyway.

Reply:
Gotcha, thanks.